Dockerfile

```diff
@@ -28,6 +29,7 @@ RUN echo "@edge http://nl.alpinelinux.org/alpine/edge/main" >>/etc/apk/repositor
 
```

Comment:
Can we remove xvfb and wkhtmltopdf@edgetesting from the Dockerfile?

Reply:
Yep! Good idea.